### ISSUE TITLE
Installation Issues

### ISSUE THREAD

**Issue by misterblonde** [aoterodelaroza/critic2#31]
The installation guidelines didn't work for me on mojave. The configure step would fail. configure --help wouldn't solve this problem either. Is this code meant to run on mac? 

Similarly on linux server, the installation steps following git clone didn't work either. 

I've emailed you about this.

Thanks

**Comment by mkhorton** [comment]
Hi @misterblonde, replying as a user of this code. I just wanted to note that I regularly compile on macOS without problems — I install the compilers/dependencies via homebrew (https://brew.sh) first, otherwise no special steps have been required for me.

**Comment by aoterodelaroza** [comment]
We discussed this over e-mail a little bit. Unfortunately, I've never used a mac so I can't really provide much help. I would very much like to make mac users' life simpler if possible - I hope that once cmake is ready to roll out, it'll help.

**Comment by ecbrown** [comment]
I was able to install on Mac OS Big Sur with Apple clang and Macports FC=gfortran-mp-10.  I used the cmake routines, and also used my own libxc and libcint.  (Also works on Ubuntu 20.04 LTS.)

**Comment by mkhorton** [comment]
@aoterodelaroza just an idea, but you could run a GitHub Action to test a compilation on macOS on GitHub's servers. I test Windows this way for another project I run because I don't have a Windows laptop, but it at least lets me know if things start failing.

**Comment by mkhorton** [comment]
Starting from this existing action might be good, https://github.com/marketplace/actions/github-action-for-homebrew 

There's a blog post here that might be helpful too, https://extrawurst.medium.com/github-actions-homebrew-%EF%B8%8F-2789ae5023fd

I'm happy to help test if it'd be helpful at all

**Comment by aoterodelaroza** [comment]
That looks like an amazing thing to have! I'll certainly take a look
in the near future because the issue of compilation of critic2 on the
macOS has been bugging me for a while now. And also I'll take up on
your offer to test it, too (thanks!).

**Comment by aoterodelaroza** [comment]
It works amazingly! I've pushed 3 workflows for testing Linux, macOS, several gfortran versions, and cmake and autoconf. Right now it's failing because I've overrun the resources github allocates to a free account, so I may look into hosting the tests myself in the near future (other than macOS). Thanks!
One more thing: are there any words of wisdom that should go in the installation section of the manual re macOS? From the workflow, it looks like all you need is to install cmake and gcc using homebrew.

**Comment by mkhorton** [comment]
> It works amazingly! [...] Right now it's failing because I've overrun the resources github allocates to a free account, so I may look into hosting the tests myself in the near future (other than macOS).

Happy to hear it :-) I didn't realise there was a limit for free accounts, but if you have an education affiliation you might be able to get upgraded. Another strategy is to only run the CI when you tag a release or something like that.

> From the workflow, it looks like all you need is to install cmake and gcc using homebrew.

That's how I've been doing it! Been working well so far.

**Comment by mkhorton** [comment]
Btw, if it's helpful I wrote a homebrew formula a long time ago for critic2, I'm not sure if it still works:

https://github.com/mkhorton/homebrew-matsci/blob/master/Formula/critic2.rb

I didn't try to get it merged into the main homebrew because it was around the time that `homebrew-science` was going away and I didn't know what the policy was going to be.

**Comment by aoterodelaroza** [comment]
OK, I have added the workflow tests and some instructions re macOS to the manual, just in case someone is confused. I'm closing this issue now. Thanks @mkhorton and everyone for the help!
